### PR TITLE
make work stealing callback time configurable

### PR DESCRIPTION
**Pull request by lr4d**
Makes it easier to benchmark #3069

### CODE REVIEW

File: distributed/stealing.py [hunk 1]
```diff
@@ -40,8 +40,14 @@ def __init__(self, scheduler):
         for worker in scheduler.workers:
             self.add_worker(worker=worker)
 
+        callback_time = parse_timedelta(
+            dask.config.get("distributed.scheduler.work-stealing-interval"),
+            default="ms",
```
Comment (TomAugspurger) [inline distributed/stealing.py]: I think you'll need something like "100ms" here.

Comment (lr4d) [reply distributed/stealing.py]: AFAI see, `default` represents the default time unit (`def parse_timedelta(s, default="seconds")`)

Comment (TomAugspurger) [reply distributed/stealing.py]: Oh, sorry I misunderstood.

**Comment by TomAugspurger** [comment]
Thanks!

**Pull request merged**